ignite/handlers/checkpoint.py

```diff
@@ -42,6 +42,9 @@ class ModelCheckpoint(object):
             in the directory 'dirname'
         create_dir (bool, optional):
             If True, will create directory 'dirname' if it doesnt exist.
+        save_as_state_dict (bool, optional):
+            If True, will save only the `state_dict` of the object specified in the dict argument,
```

Comment:
Thanks - it's a bit confusing can we change it to
```
save_as_state_dict (bool, optional):
            If True, will save only the `state_dict` of the objects specified ~~in the dict argument~~, otherwise the whole object will be saved
```

Reply:
@alykhantejani The fix is on the way. But did You really mean ~~in the dict argument~~?

Reply:
@rpatrik96 yes, remove `in the dict argument`

Reply:
@vfdev-5 Ok, thanks! I was not 100% sure.